src_web/kamaho-shokusu/config/Migrations/20260718000001_AddReportAccessToMUserInfo.php

```
@@ -0,0 +1,20 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class AddReportAccessToMUserInfo extends AbstractMigration
```

PHPStan (2.2.5)

[error] Class AddReportAccessToMUserInfo extends unknown class Migrations\AbstractMigration.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)